docs/tests/voting.md

```diff
@@ -0,0 +1,78 @@
+<!--
+   - SPDX-FileCopyrightText: 2022 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+
+# Voting with voting wizard
+
+This document explains how one can test voting via voting wizard on during all possible
+amendment periods.
+
+## Prerequisites
+
+1) Install `tezos-baking` package should on your system.
+2) Clone [local-chain repository](https://gitlab.com/morley-framework/local-chain).
+
+## Test scenario workflow
+
+1) Generate a pair of keys associated with `baker` alias:
+
+```
+sudo -u tezos tezos-client gen keys baker
+```
+
+2) In a separate terminal start a [voting scenario script](https://gitlab.com/morley-framework/local-chain#voting-scenario) from the local-chain repo.
+
+This script will provide you a path to the node config that will be used by the custom baking service.
```

Comment:
Ok, I found where I was having problems. For some reason the node couldn't use/recognise the config that was generated by the `local-chain` scenario in my home directory 🤦 Moving it to `/var/lib/tezos/` helped

Reply:
hmm, for me even path in home directory works fine :thinking: